en/flight_controller/intel_aero.md

```diff
@@ -93,3 +93,23 @@ The pinout for the Lidar Lite V3 is as follows
 ![](../../assets/hardware/Aero_I2C_splitter.JPG)
 
 ![](../../assets/hardware/Aero_LidarLite.JPG)
+
+## Using Optical Flow on the Aero
+
+The Intel Aero comes with a preinstalled optical flow binary on the compute board (Linux OS version 1.6 or higher) which enables the Aero to stably fly based on optical flow velocity estimation. In order to use optical flow, a range sensor has to be installed first (see above).
+
+To use the optical flow, run the following command in a console on the Aero compute board:
+```
+systemctl start aero-optical-flow
+```
+If you want to start the optical flow binary at boot, use
+```
+systemctl enable aero-optical-flow #use disable to undo
+```
+
+In addition, check if the following two parameters are set correctly on the flight controller.
```

Comment:
So Aero refers to the NuttX part as the "Flight controller"? If so, cool. 
I would probably word this as:

"In addition, the following parameter values should be set in the flight controller.